src/Frontend/Modules/Pages/Layout/Widgets/SubpagesDefault.html.twig

```diff
@@ -17,7 +17,7 @@
     {% block widget_body %}
       <div class="widget-body media">
         {% for subpage in widgetSubpages %}
-          <div class="media-body">
+          <div class="media-body{% if subpage.data.link_class %} {{ subpage.data.link_class }}{% endif %}">
```

Comment:
you placed this class on this a-element in the other places; why did you place it on the div here?

Reply:
When I should use the subpages I would like to have the class on the container, so you can work with the "subpage block". For example, adding a background color/image, adding a hover effect, ... in most cases I make this block fully clickable for better UX.

Reply:
I would add it by default in the fork theme to the link to keep it consistant. You can always change it in your personal projects

Reply:
Good point. Done!